rpcadaptors.go

```diff
@@ -359,12 +359,21 @@ func (b *rpcSyncMgr) LocateBlocks(locator blockchain.BlockLocator, hashStop *cha
 	return b.server.chain.LocateBlocks(locator, hashStop, maxHashes)
 }
 
-// ExistsAddrIndex returns the address index.
-//
-// This function is safe for concurrent access and is part of the
-// rpcserver.SyncManager interface implementation.
-func (b *rpcSyncMgr) ExistsAddrIndex() *indexers.ExistsAddrIndex {
-	return b.server.existsAddrIndex
+// ExistsAddress returns whether or not an address has been seen before.
+func (b *rpcSyncMgr) ExistsAddress(addr dcrutil.Address) (bool, error) {
+	if b.server.existsAddrIndex == nil {
+		return false, rpcserver.ErrExistsAddrIndexDisabled
```

Comment:
This should be `nil` rather than `false` here

Reply:
The bool? Can you set a bool to nil!??! I thought the nil value was false, as a nil value for `int` is 0, or string is an empty string.

Reply:
Whoops, my mistake, I must have been looking at the function signature for `ExistsAddresses`, this looks correct